packages/tideline/plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -55,13 +55,23 @@ class BasicsChartNoSize extends React.Component {
     }
 
     const basicsData = _.cloneDeep(tidelineData.basicsData)
+
+    // We only want the last 15 days for the dashboard
+    const now = new Date()
+    const twoWeeksAgo = new Date()
+    twoWeeksAgo.setDate(now.getDate() - 15)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏It might be nice to reuse `DEFAULT_DASHBOARD_TIME_RANGE_DAYS` from `patient-data.util.ts`, in order to have everything working if we ever change the dashboard time range again.
Also, in that spirit, `twoWeeksAgo` could be renamed `dashboardStartDate`